libafl/src/fuzzer/mod.rs

```diff
@@ -217,7 +217,7 @@ where
         // But as the state may grow to a few megabytes,
         // for now we won' and the user has to do it (unless we find a way to do this on `Drop`).
 
-        Ok(ret)
+        Ok(ret.expect("iters should not be able to be 0"))
```

Comment:
Since this is redundant, we could just use this to return the `illegal_argument` Error from above